statler_site/templates/play-list-page.html

```diff
@@ -14,16 +14,23 @@
       $(document).ready(function(){
         // Get JSON from a URL as an object
         $.getJSON(apiEndpointUrl, function(result){
-          // display the returned json
-          $('p').append(JSON.stringify(result));
+          // display the returned json dynamically
+          $.each(result, function(key, value){
```

Comment:
The first argument is more of an index than a key, correct? (This is the sort of nitpicky note you're welcome to ignore if you disagree.)